test/Microsoft.Health.Fhir.Ingest.Template.UnitTests/CodeValueFhirTemplateFactoryTests.cs

```diff
@@ -280,6 +280,20 @@ public void GivenValidTemplateJsonWithMixedMainAndComponentValues_WhenFactoryCre
                 });
         }
 
+        [Theory]
+        [FileData(@"TestInput/data_CodeValueFhirTemplateInvalid_MissingFields.json")]
+        public void GivenInvalidTemplateJsonMissingFields_WhenFactoryCreate_ThenTemplateCreated_Test(string json)
+        {
+            var templateContainer = JsonConvert.DeserializeObject<TemplateContainer>(json);
+
+            var factory = new CodeValueFhirTemplateFactory();
+
+            var template = factory.Create(templateContainer);
+            Assert.NotNull(template);
+            template.IsValid(out string errors);
+            Assert.Contains("TypeName", errors);

```

Comment:
Should test the template is functional.  A separate test that has a good template and bad template and verifies the bad template is skipped and the good template works would be a good addtition.

Reply:
Thanks for pointing out. I thought the validity of the template is binary, and was not aware of this case.

I'll update.